configs/body/3d_kpt_mview_rgb_img/voxelpose/panoptic/voxelpose_prn64x64x64_cpn80x80x20_panoptic_cam5.py

```diff
@@ -115,12 +115,11 @@
         type='V2VNet', input_channels=num_joints, output_channels=1),
     center_head=dict(
         type='CuboidCenterHead',
-        cfg=dict(
```

Comment:
The config should be consistent with the model definition, for example, the `type` of model here should be `DetectAndRegress` instead of `VoxelPose`. Check other fields that need to be modified.